test/Tester/RelationalUtilities/RelationalStorageForTesting.cs

```diff
@@ -17,11 +17,11 @@ public abstract class RelationalStorageForTesting
                 {AdoNetInvariants.InvariantNameMySql, cs => new MySqlStorageForTesting(cs)}
             };
 
-        private readonly IRelationalStorage storage;
```

Comment:
I looked into this. I think basically the issue here is that I use this mechanism to test the ADO.NET wrapper functionality itself, not Orleans constructs. So, I need access to the actual wrapper. I could instantiate anothe storage, but on the other hand, this looks like a handy way to piggy-bag on an already created one. Regarding future, I might implement some kind of sniffing functionality if the DB really is there instead of just failing the relevant DB tests, so this arrangement might serve well there too.